CI worker test_polars_struct_thread_panic_error is broken

NOW:
CI worker test `test_polars_struct_thread_panic_error` raises `RepositoryNotFoundError`: https://github.com/huggingface/dataset-viewer/actions/runs/11053126786/job/30706767804?pr=3037
```
ERROR tests/job_runners/split/test_descriptive_statistics.py::test_polars_struct_thread_panic_error - huggingface_hub.utils._errors.RepositoryNotFoundError: 404 Client Error. (Request ID: Root=1-66f560ea-0d3bd15934a45d7838ac6bb3;f1b1e4ce-2768-4f41-a5d8-f1c1c129addc)

Repository Not Found for url: https://hub-ci.huggingface.co/datasets/__DUMMY_TRANSFORMERS_USER__/test_polars_panic_error/resolve/main/test_polars_panic_error.parquet.
Please make sure you specified the correct `repo_id` and `repo_type`.
If you are trying to access a private or gated repo, make sure you are authenticated.
```

Direct cause is that the dataset has been deleted from the CI Hub: https://hub-ci.huggingface.co/datasets/__DUMMY_TRANSFORMERS_USER__/test_polars_panic_error

BEFORE:
~~CI worker test `test_polars_struct_thread_panic_error` raises `LocalEntryNotFoundError`: https://github.com/huggingface/dataset-viewer/actions/runs/11052280214/job/30704011957~~
```
ERROR tests/job_runners/split/test_descriptive_statistics.py::test_polars_struct_thread_panic_error - huggingface_hub.utils._errors.LocalEntryNotFoundError: An error happened while trying to locate the file on the Hub and we cannot find the requested files in the local cache. Please check your connection and try again or make sure your Internet connection is on.
```

~~Direct cause is a 500 server error:~~
```
E           huggingface_hub.utils._errors.HfHubHTTPError: 500 Server Error: Internal Server Error for url: https://hub-ci.huggingface.co/datasets/__DUMMY_TRANSFORMERS_USER__/test_polars_panic_error/resolve/main/test_polars_panic_error.parquet (Request ID: Root=1-66f555d0-2480d6c65b27b8e7047e6f9c;093b5eeb-e881-4c93-bafd-12965f704b66)
E           
E           Cannot read properties of undefined (reading 'cloudfront') (internal error hidden in production)
```

--- a/services/worker/tests/job_runners/split/test_descriptive_statistics.py
+++ b/services/worker/tests/job_runners/split/test_descriptive_statistics.py
@@ -6,10 +6,8 @@ from http import HTTPStatus
 from typing import Optional
 
 import pandas as pd
-import polars as pl
 import pytest
 from datasets import Dataset
-from huggingface_hub.hf_api import HfApi
 from libcommon.dtos import Priority
 from libcommon.resources import CacheMongoResource, QueueMongoResource
 from libcommon.simple_cache import upsert_response
@@ -25,7 +23,6 @@ from worker.statistics_utils import (
     ColumnType,
 )
 
-from ...constants import CI_HUB_ENDPOINT, CI_USER_TOKEN
 from ...fixtures.hub import HubDatasetTest
 from ...test_statistics_utils import (
     count_expected_statistics_for_bool_column,
@@ -332,38 +329,6 @@ def image_statistics_expected() -> dict:  # type: ignore
         "statistics": dataset_statistics,
         "partial": False,
     }
-
-
-@pytest.fixture
-def struct_thread_panic_error_parquet_file(tmp_path_factory: pytest.TempPathFactory) -> str:
-    repo_id = "__DUMMY_TRANSFORMERS_USER__/test_polars_panic_error"
-    hf_api = HfApi(endpoint=CI_HUB_ENDPOINT)
-
-    dir_name = tmp_path_factory.mktemp("data")
-    hf_api.hf_hub_download(
-        repo_id=repo_id,
-        filename="test_polars_panic_error.parquet",
-        repo_type="dataset",
-        local_dir=dir_name,
-        token=CI_USER_TOKEN,
-    )
-    return str(dir_name / "test_polars_panic_error.parquet")
-
-
-def test_polars_struct_thread_panic_error(struct_thread_panic_error_parquet_file: str) -> None:
-    from polars import (
-        Float64,
-        List,
-        String,
-        Struct,
-    )
-
-    df = pl.read_parquet(struct_thread_panic_error_parquet_file)  # should not raise
-    assert "conversations" in df
-
-    conversations_schema = List(Struct({"from": String, "value": String, "weight": Float64}))
-    assert "conversations" in df.schema
-    assert df.schema["conversations"] == conversations_schema
 
 
 @pytest.mark.parametrize(

